chart/values.yaml

```diff
@@ -159,6 +159,22 @@ promlens:
   loadBalancer:
     enabled: false
 
```

Comment:
We should put some comments about this being experimental and also highlight which components should be disabled if this is enabled. In fact, I might have a kube_prometheus_values.yml file in this dir that would serve as an example of what that would look like